### ISSUE TITLE
STAR Alignment - FileNotFoundError: [Errno 2] No such file or directory (...) GeneFull_Ex50pAS/raw/barcodes.tsv

### ISSUE THREAD

**Issue by plhm** [odomlab2/sci-rocket#20]
Hey, J0bbie.

The program is crashing post-STAR run. As you can see, it is not finding the "barcodes.tsv" file it expects to find. I ran STAR by hand using the following command:

`/path/to/.snakemake/conda/964fc63d69a335d6c61b4a8b3dc63b7e_/bin/STAR-avx2 --soloFeatures GeneFull --soloCellReadStats Standard --soloMultiMappers EM --outSAMmultNmax 1 --outSAMstrandField intronMotif --outFilterScoreMinOverLread 0.33 --outFilterMatchNminOverLread 0.33 --outSAMunmapped Within --outSAMattributes NH HI AS nM NM MD jM jI MC ch XS CR UR GX GN sM --genomeDir resources/index_star/Drerio --runThreadN 12 --readFilesIn path/to/fastp/SAMPLE_R2.fastq.gz SAMPLE_sci-test/fastp/SAMPLE_R1.fastq.gz --readFilesCommand zcat --soloType CB_UMI_Complex --soloCBmatchWLtype Exact --soloCBposition 0_0_0_9 0_10_0_19 0_20_0_29 0_30_0_39 --soloUMIposition 0_40_0_47 --soloCBwhitelist SAMPLE_sci-test/demux_reads/SAMPLE_sci-test_whitelist_p7.txt SAMPLE_sci-test/demux_reads/SAMPLE_sci-test_whitelist_p5.txt SAMPLE_sci-test/demux_reads/SAMPLE_sci-test_whitelist_ligation.txt SAMPLE_sci-test/demux_reads/SAMPLE_sci-test_whitelist_rt.txt --soloCellFilter CellRanger2.2 12000 0.99 10 --outSAMtype BAM SortedByCoordinate --outFileNamePrefix SAMPLE_sci-test/alignment/SAMPLE_SPECIES_`

And noticed that rather than producing the "GeneFull_Ex50pAS/raw/barcodes.tsv" path, it only produced a "GeneFull/raw/barcodes.tsv" path. Should it have produced both paths, and I am missing one for whatever reason, or can I tweak the code to look into the "GeneFull/raw/barcodes.tsv" file instead?

Oh, and STAR seemed to have ran smoothly:
>Dec 22 07:40:37 ..... started STAR run
>Dec 22 07:40:37 ..... loading genome
>Dec 22 07:40:51 ..... started mapping
>Dec 22 07:41:04 ..... finished mapping
>Dec 22 07:41:05 ..... started Solo counting
>Dec 22 07:41:05 ..... finished Solo counting
>Dec 22 07:41:05 ..... started sorting BAM
>Dec 22 07:41:06 ..... finished successfully

**Comment by J0bbie** [comment]
Hi Pietro,

We now use the `--soloFeatures GeneFull_Ex50pAS` instead of `--soloFeatures GeneFull`.
We switched as this also gives output on the number of exonic/intronic/intergenic reads etc. 

This indeed changes which folder it outputs.

If you'd like to run it on the old settings, swapping`GeneFull_Ex50pAS` to `GeneFull` will make it work. But likely the downstream sci-dash step will fail as it's expecting some columns in the soloCellReadStats file.

I had tweaked the [`example_config.yaml`](https://github.com/odomlab2/sci-rocket/blob/main/workflow/examples/example_config.yaml) to reflect this, but this was indeed easy to miss. I've now hardcoded some of the required parameters directly into the rule instead.

Hope this helps! Sorry for all the extra work.
